custom_components/test/fuzzing/fuzzer_utils/GrammarFuzzer.py

```diff
@@ -136,17 +137,17 @@ def fuzz_min_cost(self, grammar: Grammar, start_symbol: Element) -> str:
         return self.__compose_min_cost(start_symbol, cost_grammar)
 
     def __compose_max_cost(
-        self,
-        head: Element,
-        given_cost_grammar: Annotated_Grammar,
-        applications: int,
-        max_applications: int,
+            self,
+            head: Element,
+            given_cost_grammar: Annotated_Grammar,
+            applications: int,
+            max_applications: int,
     ) -> str:
         """TODO: Add summary"""
```

Comment:
Some function descriptions are missing.

Reply:
ref to [7d2de52](https://github.com/ThorbenCarl/PyLoxone/pull/18/commits/7d2de52eafbe877af951ea7f5e2d3affa08d3395) I still found 7 `"""TODO: Add summary"""` 😉